src/prisma/schema.prisma

```diff
@@ -11,7 +11,25 @@ generator client {
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
-  email    String @unique
+  id       Int      @id @default(autoincrement())
+  email    String   @unique
   password String
+  groups   Member[]
```

Comment:
Hey Team, again, thank you for the effort put into this PR. 

It would be great to add createdAt and updatedAt datetime as best practices. 

Thanks